debian/control

```diff
@@ -0,0 +1,20 @@
+Source: alire
+Section: devel
+Priority: extra
+Maintainer: Stephane Carrez <Stephane.Carrez@gmail.com>
+Build-Depends: debhelper (>= 10), autotools-dev, gnat,
+ gprbuild (>= 2015-2),
+Standards-Version: 4.1.4
+Homepage: https://github.com/alire-project
+Vcs-Git: https://github.com/alire-project/alire.git
+Vcs-Browser: https://github.com/alire-project/tree/master
+
+Package: alire
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: Ada package manager.
+ A catalog of ready-to-use Ada libraries plus a command-line tool
+ (`alr`) to obtain, build, and incorporate them into your own projects.
+ It aims to fulfill a similar role to Rust's `cargo` or OCaml's `opam`.
+
+
```

Comment:
lintian reports the following rule violations for this source package:
```
E: alire source: missing-python-build-dependency
W: alire source: package-needs-versioned-debhelper-build-depends 11
W: alire source: useless-autoreconf-build-depends autotools-dev
```

...and these for the amd64 binary:
```
E: alire source: missing-python-build-dependency
W: alire: bad-whatis-entry usr/share/man/man1/alr-action.1.gz
W: alire: bad-whatis-entry usr/share/man/man1/alr-build.1.gz
W: alire: bad-whatis-entry usr/share/man/man1/alr-clean.1.gz
W: alire: bad-whatis-entry ... use --no-tag-display-limit to see all (or pipe to a file/program)
W: alire: groff-message usr/share/man/man1/alr-action.1.gz 6: warning: macro 'SH' not defined
W: alire: groff-message usr/share/man/man1/alr-build.1.gz 6: warning: macro 'SH' not defined
W: alire: groff-message usr/share/man/man1/alr-clean.1.gz 6: warning: macro 'SH' not defined
W: alire: groff-message ... use --no-tag-display-limit to see all (or pipe to a file/program)
W: alire source: package-needs-versioned-debhelper-build-depends 11
W: alire: priority-extra-is-replaced-by-priority-optional
W: alire source: useless-autoreconf-build-depends autotools-dev
```

Reply:
I've fixed some of your remarks and warnings reported by `lintian`.

I don't pretend this Debian package is perfect and if you want to propose something else or if you could help in having such Debian package integrated officially, it would be good. Once it is in a suitable good shape, we can probably reach the debian-ada mailing list and talk with Nicolas who handles several Debian Ada packages there.